data/themes/elarun/Main.qml

```diff
@@ -200,18 +200,24 @@ Rectangle {
                         }
                     }
 
-                    Text {
-                        id: time_label
+                    Rectangle {
+                        id: container
+                        property date dateTime: new Date()
+                        property color color: "#0b678c"
                         anchors.right: parent.right
-                        anchors.bottom: parent.bottom
 
-                        text: Qt.formatDateTime(new Date(), "dddd, dd MMMM yyyy HH:mm AP")
```

Comment:
what was wrong with the previous version is here, by using a Timer that updates a dateTime property you should be able to update the clock without moving this Text here into a container